src/handler.ts

```diff
@@ -1,34 +1,30 @@
-import { Router, Method, Params } from 'tiny-request-router'
+import { Router, Method, RouteMatch } from 'tiny-request-router'
 import xmlBodyParser from './utils/xmlBodyParser'
 import autodiscover from './autodiscover'
-import settings from './settings'
 import mobileconfig from './mobileconfig'
 import autoconfig from './autoconfig'
+import settings from './settings'
 
 const router = new Router()
-// router.get('/worker', async () => new Response('Hi from worker!'))
-// router.get(
-//   '/hello/:name',
-//   async (params: Params) => new Response(`Hello ${params.name}!`),
-// )
-// router.post('/test', async () => new Response('Post received!'))
 
-// ******  Autodiscover Routes
+// *** Auto Discover Routes
 const autodiscoverParsedXML = xmlBodyParser(autodiscover(settings))
-// Outlook
+
+// Microsoft Outlook
 router.get('/autodiscover/autodiscover.xml', autodiscoverParsedXML)
 router.post('/autodiscover/autodiscover.xml', autodiscoverParsedXML)
 router.get('/Autodiscover/Autodiscover.xml', autodiscoverParsedXML)
 router.post('/Autodiscover/Autodiscover.xml', autodiscoverParsedXML)
-// Thunderbird
+
+// Mozilla Thunderbird
 router.get('/mail/config-v1.1.xml', autoconfig(settings))
-// iOS / Apple Mail (/email.mobileconfig?email=username@domain.com or /email.mobileconfig?email=username)
+
+// Apple iOS / Mail (/email.mobileconfig?email=username@domain.com or /email.mobileconfig?email=username)
 router.get('/email.mobileconfig', mobileconfig(settings))
 
 export default (request: Request) => {
   const { pathname } = new URL(request.url)
-
-  const match = router.match(request.method as Method, pathname)
+  const match: RouteMatch<any> | null = router.match(request.method as Method, pathname)
```

Comment:
Ah yes.  This is helpful, thank you.  It wasn't complaining in VSCode, which is strange.